src/main/java/io/vertx/mqtt/MqttEndpoint.java

```diff
@@ -242,6 +292,17 @@
   @Fluent
   MqttEndpoint publishReleaseHandler(Handler<Integer> handler);
 
+  /**
+   * Set the pubrel handler on the MQTT endpoint. This handler is called when a PUBREL
+   * message is received by the remote MQTT client
+   *
+   * @param handler the handler
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  MqttEndpoint publishReleaseMessageHandler(Handler<MqttPubRelMessage> handler);
```

Comment:
I'm wondering if we should not bame it `PubRelMesssageHandler` for consistency

Reply:
What should we do with `MqttEndpoint publishReleaseHandler(Handler<Integer> handler);` then?

Reply:
right this is a good point.